### PR TITLE
New method for greeting and method call.

### PR DESCRIPTION
**Pull request by PhallyPhal**
New function added, justinsGreeting and added a call to it in the builder.

### CODE REVIEW

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -33,4 +35,7 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+  private String justinSaysHello() {
+    return "Justin says 'Oh...well hello there!'\n";
```
Comment (DanteMillerDS) [inline src/main/java/hellos/Hellos.java:65]: Too many dots lol

**Review by DanteMillerDS — DISMISSED** [review]

File: src/main/java/hellos/Hellos.java (hunk 1 above)
Comment (PhallyPhal) [reply src/main/java/hellos/Hellos.java:65]: My bad lol.

**Review by halleypaulson — DISMISSED** [review]
U get my stamp of approval

**Review by halleypaulson — DISMISSED** [review]
looks good

**Review by NicMcPhee — DISMISSED** [review]
Looks great, dot's and all. 😜

**Review by NicMcPhee — APPROVED** [review]

**Review by helloworld12321 — APPROVED** [review]
Looks very nice! =)

**Pull request merged**